abdulrahman/style.css

```diff
@@ -0,0 +1,100 @@
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+body {
+    background-color: rgb(248, 217, 248);
+    line-height: 1.5;
+    word-spacing: 1.4px;
+    font-family: 'Roboto', sans-serif;
+    height: 100%;
+    color: black;
+    position: relative;
+}
+
+.card__container {
+    display: flex;
+    width: 60%;
+    height: auto;
+    margin: 300px auto;
```

Comment:
The width and margin of the auto will center your container thus you don't need the position and left attributes.
Also, when ever you have a margin above 100px, you MIGHT be doing the wrong thing. That margin is too much